test/test_browser.py

```diff
@@ -5443,6 +5444,7 @@ def test_wasmfs_fetch_backend(self, args):
   @parameterized({
     '': (['-pthread', '-sPROXY_TO_PTHREAD'],),
     'jspi': (['-Wno-experimental', '-sASYNCIFY=2'],),
+    'jspi_wasm_bigint': (['-Wno-experimental', '-sASYNCIFY=2', '-sWASM_BIGINT'],),
```

Comment:
On the main branch, this test fails with WASM_BIGINT. So this code was also negatively affected by the same bug.

Reply:
I see yes.  I think that bug could be fixed by changing `if MEMORY64` to `if WASM_BIGINT` in the `sigToWasmTypes`.. although this PR will also fix it.

Reply:
Yeah, the only issue with that is that it is out of sync with the no `WebAssembly.Function` case. Either `addFunction` should have logic to handle no-wasm-bigint both with and without `WebAssembly.Function` or neither with nor without `WebAssembly.Function`. Either way would make sense. But the current situation where there is buggy handling only in the `WebAssembly.Function` case is problematic. 

Also, I think the other solution would mean I have to change my downstream code... which wouldn't be the end of the world.

Reply:
I think fixing the attached `.sig` attributes is probably the way to go.